src/integration-test/java/com/commercetools/sync/integration/externalsource/taxcategories/TaxCategorySyncIT.java

```diff
@@ -221,116 +216,4 @@ private SphereClient buildClientWithConcurrentModificationUpdate() {
 
         return spyClient;
     }
```

Comment:
Similar to another two test cases for `ConcurrentModificationException`, this mocking is also required to move to TaxCategorySyncTest

Reply:
https://github.com/commercetools/commercetools-sync-java/commit/30bc2438b9e4c3d18e635410535472734c0b862d